api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -304,6 +306,43 @@
    * @since 0.1.0
    */
   interface Builder {
+    /**
+     * Sets the parent {@code Span} to use. If not set, {@code Tracer.getCurrentSpan()} will be used
+     * as parent.
+     *
+     * <p>This <b>must</b> be used to create a {@code Span} when manual Context propagation is used
+     * OR when creating a root {@code Span} with a {@code null} parent or a parent with an invalid
+     * {@link SpanContext}.
+     *
+     * <p>Observe this is the preferred method when the parent is a {@code Span} created within the
+     * process. Using its {@code SpanContext} as parent remains as a valid, albeit inefficient,
+     * operation.
+     *
+     * <p>If called multiple times, only the last specified value will be used.
+     *
+     * @param parent the {@code Span} used as parent.
+     * @return this.
+     * @since 0.1.0
+     */
+    Builder setParent(@Nullable Span parent);
+
+    /**
+     * Sets the parent {@link SpanContext} to use. If not set, {@code Tracer.getCurrentSpan()} will
```

Comment:
I would prefer keeping the main javadoc for  `Builder setParent(@Nullable Span parent);` , referencing it from here and just adding what is different (if there is something different).